nixos/modules/services/web-apps/nextcloud.nix

```diff
@@ -45,6 +45,12 @@ let
   inherit (config.system) stateVersion;
 
 in {
+
+  imports = [
+    ( mkRemovedOptionModule [ "services" "nextcloud" "nginx" "enable" ]
+      "The nextcloud module dropped support for other webservers than nginx.")
```

Comment:
That's wrong: you can still use anything else (in fact now it's even easier than before!) by setting `services.nginx.enable = false`.

Reply:
That's not entirely true. One also has to change the `listen.owner` and `listen.group` of phpfpm to something else than nginx and also add the user of the new webserver to `users.groups.nextcloud.members`. Should I add an example for disabling nginx to the deprecation text?

>We should mention this in the release notes. The rollout of the new config can take quite long due to the recursive chgrp

As I explained in my last comment, The rollout won't take any longer than the normal startup of the previous implementation. Therefore I think it's unnecessary to state anything in the release notes. Do you agree?

Reply:
> Should I add an example for disabling nginx to the deprecation text?

Yes, please. But as this is possible (with a few overrides though), I'd also rephrase the message above.

> Therefore I think it's unnecessary to state anything in the release notes. Do you agree?

As mentioned above, I only tested this against a temporary environment. As I'm always slightly anxious when fiddling around with those kinds of scripts (touching user data) in NixOS, I'd prefer to test this against live-data first. I'll hopefully be able to do this tomorrow (with the fat library I'm hosting in my personal Nextcloud :P). After that I think that this should be good to go :)

Reply:
Tested against my instance, didn't have any problems.